templates/cases/case_read.haml

```diff
@@ -78,7 +78,7 @@
         .timeline{ ng-controller:"CaseTimelineController", ng-init:"init()" }
           .timeline-event.clearfix{ ng-repeat:"event in timeline | reverse", ng-class:'{ "timeline-action": (event.type == "A"), "timeline-incoming": (event.type == "I"), "timeline-outgoing": (event.type == "O") }' }
             .event-time
-              [[ event.time | autodate ]]
+              <cp-date-tooltip time=event.time position="top-right" />
```

Comment:
Is there a reason for omitting the quotes here? Could be just a bug in our HAML implementation that it works without them.